demos/demo1-chat-heads/Demo.jsx

```diff
@@ -1,59 +1,222 @@
-import React from 'react';
-import {StaggeredMotion, spring, presets} from '../../src/react-motion';
-import range from 'lodash.range';
+import React from "react";
+import {
+  StaggeredMotion,
+  spring,
+  Motion,
+  presets
+} from "../../src/react-motion";
+import range from "lodash.range";
+import Slider from "./Slider";
 
-export default class Demo extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {x: 250, y: 300};
-  };
+const options = [
```

Comment:
To reduce the amount of code, maybe it's worthy to make a reusable fn for generating all the `options` available?

Like (you can choose any value separator you need, I picked `_` as it's more readable on longer strings):

`makeOptions = (options) => options.map(option => ({ description: option, value: option.replace(' ', '_').toLowerCase() }))`

So your options become a concise `const options = makeOptions([ 'Constant', 'Linear', 'InverseLinear' ]);`